plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergConnectorSmokeTest.java

```diff
@@ -35,7 +38,13 @@
     protected QueryRunner createQueryRunner()
             throws Exception
     {
-        return createIcebergQueryRunner(ImmutableMap.of(), ImmutableMap.of(), REQUIRED_TPCH_TABLES);
+        return createIcebergQueryRunner(ImmutableMap.of(), ImmutableMap.of(), Optional.empty(), this::initializeSchema);
+    }
+
+    protected void initializeSchema(QueryRunner queryRunner)
+    {
+        queryRunner.execute("CREATE SCHEMA tpch");
+        copyTpchTables(queryRunner, "tpch", TINY_SCHEMA_NAME, queryRunner.getDefaultSession(), REQUIRED_TPCH_TABLES);
```

Comment:
This is the typical case. In the typical case, an additional initializeSchema method should not be needed.
`createIcebergQueryRunner` should still accept `REQUIRED_TPCH_TABLES`